src/components/Sum.jsx

```diff
@@ -1 +1,22 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum({ a, b }) {
+  return (
+    <>
```

Comment:
remove unnecessary fragment
Also, answer me please, why we need a fragment?

Reply:
Fragment use for return few tags

Reply:
Great)
So, you don't have a few tags here, yeah?

Reply:
I just return one tag, so don't need a wrapper tag